fx.go

```diff
@@ -18,4 +18,150 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package fx // import "go.uber.org/fx"
```

Comment:
You lost the ``// import "go.uber.org/fx"` comment which is important to keep for canonical import paths